js/foundation.slider.js

```diff
@@ -1,7 +1,14 @@
 'use strict';
 
-!function($) {
+import $ from 'jquery';
+import { Keyboard } from './foundation.util.keyboard';
+import { Move } from './foundation.util.motion';
+import { GetYoDigits, rtl } from './foundation.util.core';
+let Rtl = rtl;
```

Comment:
It's valid 👍  see <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import>